media-libs/assimp/assimp-5.0.1.recipe

```diff
@@ -59,19 +63,24 @@ defineDebugInfoPackage assimp$secondaryArchSuffix \
 BUILD()
 {
 	mkdir -p build
-	cd build
```

Comment:
This won't work if you use cmake .. after this without switching to build

Reply:
make **-C build** $jobArgs won't work either? ?_?

Reply:
You are calling that after **cmake * ..** :)

Reply:
oh s*it! -_- going to fix immediately